src/main.rs

```diff
@@ -308,10 +309,15 @@ fn main() {
         SubCommand::with_name( "deploy" )
             .about( "Deploys your project so that its ready to be served statically" );
 
+    let mut prepare_emscripten_subcommand =
+        SubCommand::with_name( "prepare-emscripten" )
+            .about( "Prepares and installs emscripten" );
```

Comment:
`"Fetches and installs prebuilt Emscripten packages"` would be a little better as a description, I think.